src/pages/App.tsx

```diff
@@ -307,6 +310,9 @@ export default function App() {
                     <Route path={`${APP_PATHS.GRANT_PROGRAMS}`} element={<GrantProgramPage />} />
                     <Route path={`${APP_PATHS.GRANT_PROGRAMS}/:slug`} element={<GrantProgramPage />} />
 
+                    <Route path={`${APP_PATHS.ELASTIC_LEGACY}`} element={<ElasticLegacy />} />
+                    <Route path={`elastic-swap`} element={<ElasticSwap />} />
```

Comment:
We should add this to APP_PATHS

Reply:
dont use anywhere in code, just a page to test and support QA swap directly through elastic pool